### DISCUSSION TITLE
夏莱联邦事务局入驻绳网

### DISCUSSION THREAD

**Discussion by schalesensei843** [share121/inter-knot#1992] [其他]
![1691880639472](https://github.com/user-attachments/assets/a9179397-21e7-4256-b9b2-6cb7bf74b239)
各位绳匠在闲暇时间可以来学园都市奇窝托斯旅游参观

**Comment by blqdswkl** [comment]
嗯！月雪宫子，可爱！嗯嗯嗯！
